gcf_data_mapper/read.py

```diff
@@ -0,0 +1,62 @@
+import csv
+import json
+from typing import Any, Optional, Union
+
+import click
+
+
+def read_csv(file_path: str) -> list[dict[str, Any]]:
+    """
+    Reads a csv file and returns a list of dictionaries
+
+    :param str: a file path to the csv file
+    :return list: A list of strings
```

Comment:
This doesn't match the return type in the function signature

Reply:
oops